api/db/migrations/20251110104207_create-table-sco-blocked-access-dates-table.js

```diff
@@ -0,0 +1,30 @@
+const TABLE_NAME = 'sco_blocked_access_dates';
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+const up = async function (knex) {
+  await knex.schema.createTable(TABLE_NAME, function (table) {
+    table
+      .string('scoOrganizationType', 255)
```

Comment:
Le nommage m'induit un peu en erreur, ce champ ne porte pas un organization type, mais un organization **tag**
Les dates de fermeture utilisent bien les tags, qui sont le champ `name` de la table en BDD `tags`

D'ailleurs proposition : plutôt que de dupliquer les valeurs, pourquoi ne pas faire une foreign key sur la table tags ? C'est ainsi par exemple que la table `organization-tags` fonctionne

Ou si la proposition ne séduit pas alors aligner le vocabulaire sur le nom de colonne


```suggestion
      .string('scoOrganizationTagName', 255)
```

Reply:
A propos de la table `tags` : au départ elle permettait d’identifier plus facilement certaines organisations pour activer certaines fonctionnalités particulières. Puis au fur et à mesure cette fonctionnalité a été aussi utilisée à des fins de stats... et actuellement elle contient plus de 200 tags, qui appartiennent à des domaines très différents.
 Il se trouve qu'elle contient les valeurs qui nous intéressent, mais je pense qu'elle couvre des champs trop vastes et divers pour être utilisée dans le cadre de cette PR.